docs/manual/common/guide/production/LightbendPlatform.md

```diff
@@ -1,8 +1,8 @@
-# Using Lightbend Enterprise Suite with Lagom
+# Using Lightbend Platform with Lagom
 
-Lagom is part of the [Lightbend Platform](https://www.lightbend.com/lightbend-platform) — an application development solution for building Reactive systems on the JVM powered by an open source core and the commercial [Enterprise Suite](https://www.lightbend.com/products/enterprise-suite). Lagom services can be developed, deployed and run in production using only open source components, however we recommend considering the advanced features provided by Enterprise Suite.
+Lagom is part of the [Lightbend Platform](https://www.lightbend.com/lightbend-platform) — an operationally robust solution for deploying Reactive Microservices. It gives you the ability to infuse your applications with intelligence based on real-time streaming and Machine Learning (ML). Lagom services can be developed, deployed and run in production using only open source components. However, we recommend considering the advanced features provided by Lightbend Platform.
 
-This page highlights the Enterprise Suite features that are especially useful for most Lagom users, but there are many more. You can find an overview on the [Lightbend web site](https://www.lightbend.com/products/enterprise-suite) or [contact Lightbend](https://www.lightbend.com/contact) to learn more.
+This page highlights the Lightbend Platform features that are especially useful for most Lagom users, but there are many more. You can find an overview on the [Lightbend web site](https://www.lightbend.com/lightbend-platform) or [contact Lightbend](https://www.lightbend.com/contact) to learn more.
 
 ## Akka Commercial Addons
```

Comment:
If we're waiting to change "Akka Commercial Addons" in this section, that's ok. When we do make that modification, I think it's important to clearly divide between "Akka Resilience Enhancements" and "Akka Persistence Enhancements".

In the interim:
- Could we add a one sentence description for each of the commercial pieces (similar to this document: https://docs.google.com/document/d/1WuIMDWADDCZbwpoq9lIUbpxXMLD7PGSsM6suO6GJaHs/edit?usp=sharing)?
- Should there be a new bullet for Kubernetes Lease?

Reply:
Done (both)